cime_config/user_nl_blom_outvol_reduced

```diff
@@ -0,0 +1,534 @@
+  LYR_DIFVHO   =  0,   0
+  LYR_DIFVMO   =  0,   0
+  LYR_DIFVSO   =  0,   0
+  LYR_DP@diaphy=  4,   4
+  LYR_DZ       =  0,   0
+  LYR_SALN     =  0,   0
+  LYR_TEMP     =  0,   0
+  LYR_TRC      =  0,   0
+  LYR_UFLX     =  0,   0
+  LYR_UTFLX    =  0,   0
+  LYR_USFLX    =  0,   0
+  LYR_UMFLSM   =  0,   0
+  LYR_UMFLTD   =  0,   0
+  LYR_USFLSM   =  0,   0
+  LYR_UTFLTD   =  0,   0
+  LYR_UTFLLD   =  0,   0
+  LYR_UTFLSM   =  0,   0
+  LYR_USFLTD   =  0,   0
+  LYR_USFLLD   =  0,   0
+  LYR_UVEL     =  0,   0
+  LYR_VFLX     =  0,   0
+  LYR_VTFLX    =  0,   0
+  LYR_VSFLX    =  0,   0
+  LYR_VMFLSM   =  0,   0
+  LYR_VMFLTD   =  0,   0
+  LYR_VTFLTD   =  0,   0
+  LYR_VTFLLD   =  0,   0
+  LYR_VSFLSM   =  0,   0
+  LYR_VSFLTD   =  0,   0
+  LYR_VSFLLD   =  0,   0
+  LYR_VTFLSM   =  0,   0
+  LYR_VVEL     =  0,   0
+  LYR_WFLX     =  0,   0
+  LYR_WFLX2    =  0,   0
+  LYR_PV       =  0,   0
+  LYR_TKE      =  0,   0
+  LYR_GLS_PSI  =  0,   0
+  LYR_IDLAGE   =  0,   0
+
+! Only temperature and salinity are written monthly
+
+  LVL_BFSQ     =  0,   4
+  LVL_DIFDIA   =  0,   0
+  LVL_DIFINT   =  0,   0
+  LVL_DIFISO   =  0,   0
+  LVL_DIFVHO   =  0,   0
+  LVL_DIFVMO   =  0,   0
+  LVL_DIFVSO   =  0,   0
+  LVL_DZ       =  4,   4
+  LVL_SALN     =  4,   4
+  LVL_TEMP     =  4,   4
+  LVL_TRC      =  0,   0
+  LVL_UFLX     =  0,   4
+  LVL_UTFLX    =  0,   4
+  LVL_USFLX    =  0,   4
+  LVL_UMFLSM   =  0,   0
+  LVL_UMFLTD   =  0,   0
+  LVL_USFLSM   =  0,   0
+  LVL_UTFLTD   =  0,   0
+  LVL_UTFLLD   =  0,   0
+  LVL_UTFLSM   =  0,   0
+  LVL_USFLTD   =  0,   0
+  LVL_USFLLD   =  0,   0
+  LVL_UVEL     =  0,   4
+  LVL_VFLX     =  0,   4
+  LVL_VTFLX    =  0,   4
+  LVL_VSFLX    =  0,   4
+  LVL_VMFLSM   =  0,   0
+  LVL_VMFLTD   =  0,   0
+  LVL_VSFLSM   =  0,   0
+  LVL_VTFLSM   =  0,   0
+  LVL_VTFLTD   =  0,   0
+  LVL_VTFLLD   =  0,   0
+  LVL_VSFLTD   =  0,   0
+  LVL_VSFLLD   =  0,   0
+  LVL_VVEL     =  0,   4
+  LVL_WFLX     =  0,   4
+  LVL_WFLX2    =  0,   0
+  LVL_PV       =  0,   4
+  LVL_TKE      =  0,   4
+  LVL_GLS_PSI  =  0,   0
+  LVL_IDLAGE   =  0,   4
+  
+  MSC_MHFSM    =  4,   4
+  MSC_MMFLXL   =  4,   4
+  MSC_MMFLXD   =  4,   4
+  MSC_MMFSMD   =  4,   4
+  MSC_MMFSML   =  4,   4
+  MSC_MMFTDL   =  4,   4
+  MSC_MMFTDD   =  4,   4
+  MSC_MHFLX    =  4,   4
+  MSC_MHFTD    =  4,   4
+  MSC_MHFLD    =  4,   4
+  MSC_MSFLX    =  4,   4
+  MSC_MSFSM    =  4,   4
+  MSC_MSFTD    =  4,   4
+  MSC_MSFLD    =  4,   4
+  MSC_VOLTR    =  4,   4
+  MSC_MASSGS   =  4,   4
+  MSC_VOLGS    =  4,   4
+  MSC_SALNGA   =  4,   4
+  MSC_TEMPGA   =  4,   4
+  MSC_SSSGA    =  4,   4
+  MSC_SSTGA    =  4,   4
+
+! -----------------------------------------------
+! iHAMOCC ouput fields
+! -----------------------------------------------
+
+  GLB_FNAMETAG@diabgc  = 'hbgcm','hbgcy'
+  GLB_AVEPERIO@diabgc  = 30,365
+  GLB_FILEFREQ@diabgc  = 30,365
+  GLB_COMPFLAG@diabgc  = 0, 0
+  GLB_NCFORMAT@diabgc  = 1, 1
+  GLB_INVENTORY        = 1, 0
```

Comment:
Can we have inventory output as well on a yearly basis?

Reply:
Sure, but what do you mean by "switch on the monitoring files"?

Reply:
It would be good to have the netcdf-monitoring files - at least yearly - hence: 
```
GLB_INVENTORY = 1,2
```
which should switch on the monitoring files for the yearly monitoring output (>1) - maybe I should just do so for the current master as well?